### DISCUSSION TITLE
Repl or codesandbox

### DISCUSSION THREAD

**Discussion by ghost** [itzg/docker-minecraft-server#2254] [Q&A]
Is there an easy way to make a server on replit or codesandbox (preferably the latter)

**Comment by itzg** [comment]
Probably not, but you'll have to ask them. 

As for the latter, [they recently (Jan 2023) added "docker support"](https://codesandbox.io/blog/introducing-docker-support-in-codesandbox); however, that's for editing your own Dockerfile.

**Reply by ghost** [reply]
A Minecraft server?

**Reply by itzg** [reply]
Yes, I already understood you're asking about a Minecraft server. It looks like the answer is no, you cannot run a Minecraft server on those providers.

**Reply by ghost** [reply]
Oh ok
